schemas/context/personalidentifiers.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/personalidentifiers",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Personal identifiers",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Condensed, normalized encapsulation of personal identifiers.",
+  "definitions": {
+    "personalidentifiers": {
```

Comment:
Aren't these defined elsewhere?  Why are we duplicating them here?

Reply:
These fields are there as complete schema objects like as follows : 
https://github.com/adobe/xdm/blob/master/schemas/context/emailaddress.schema.json
https://github.com/adobe/xdm/blob/master/schemas/context/phonenumber.schema.json

If they are of type string, then they are not together under one schema.

But we need these to be as of type String in one schema so that the personal identifiers makes sense . And in future, if more such personal identifier fields get introduced, they can also get introduced as standard fields in this schema.

Reply:
Then you can't use the same name, since the name is already taken using a different format.

I would make these `xdm:emailString` and `xdm:phonenumberString` then

Reply:
Renamed the fields. The new name has no reference in the repository